client/app/routes/collection/add.js

```diff
@@ -255,45 +304,198 @@ export default Ember.Route.extend({
+                parameters: {
+                    widget_object: 'edit_authors_section_widget'
+                },
+                then: '7231cc6f-b861-439e-a628-a01fe0a20587'
+            }, {
+                id: '1d4bcab7-c454-450d-95c5-113d89121f89',
+                type: 'create_widget',
+                args: {
+                    widget_component: 'button-widget',
+                    section: 'submit',
+                    description: 'Submit',
+                    disabled: 'true',
+                    css_classes: ['submit_button', 'btn-lg', 'btn-success'],
+                    action_id: 'afa2e526-ea5f-47f7-a99d-459536e24fd1'
+                },
+                conditions: [{
+                    all: [{
+                        parameter: 'submit_button_widget',
+                        state: 'undefined'
+                    }]
+                }],
+                output_parameter: 'submit_button_widget'
+            }, {
+                id: 'afa2e526-ea5f-47f7-a99d-459536e24fd1',
+                type: 'browserAlert',
+                args: {
+                    alertString: 'FORM SUBMITTTED'
+                }
+            }, {
+                id: 'a58fcc41-ed0c-43ab-a765-d83cb36f6a9b',
+                type: 'saveParameter',
+                args: {
+                    updated_parameter: {
+                        state: ['enabled']
+                    }
+                },
+                parameters: {
+                    parameter: 'submit_button'
+                },
+                conditions: [{
+                    all: [{
+                        parameter: 'preprint_file_data',
+                        state: 'defined',
+                    }, {
+                        parameter: 'submit_button',
+                        state: 'disabled'
+                    }, {
+                        parameter: 'preprint_file_url',
+                        state: 'defined'
+                    }, {
+                        parameter: 'selected_subjects',
+                        state: 'defined',
+                    }, {
+                        parameter: 'basic_info',
+                        state: 'defined'
+                    }, {
+                        parameter: 'authors_list',
+                        state: 'defined'
+                    }]
+                }],
+                output_parameter: 'null',
+                then: '60edeef7-189c-4189-b44a-157b79ce88ef'
+            }, {
+                id: '60edeef7-189c-4189-b44a-157b79ce88ef',
+                type: 'enableWidget',
+                parameters: {
+                    widget_object: 'submit_button_widget'
+                },
             }]
-
         };
     },
 
     setupController(controller, model) {
 
         // Set up state defined on the model.
         controller.set('sections', model.sections);
-        controller.set('parameters', model.initial_parameters);
+        controller.parameters = model.initial_parameters
```

Comment:
I think there was something where set occurs in the next run loop, but this needed to happen immediately, i'll double check